frontend/src/components/GeocacheList/GeocacheList.scss

```diff
@@ -13,4 +13,8 @@
     background-color: white;
     padding: 10px;
     color: black;
+}
+
+p{
+    margin-bottom: 0 !important;
```

Comment:
We should try to avoid using these !important statements. They usually make things hard to debug, since it overrides everything. Propably doesn't matter too much here but something to keep in mind in the future.